### PR TITLE
5581 fixed month label not set on first enabled date of the month

### PR DESCRIPTION
**Pull request by edinfor**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Use a set to store all months label that needs to be set, and remove once it's set on the first enabled date.

**Related github/jira issue (required)**:
Fixes https://github.com/infor-design/enterprise/issues/5581

**Steps necessary to review your pull request (required)**:

1. go to calendar in range: http://localhost:4000/components/calendar/test-range-in-month.html
2. go to app/views/components/calendar/test-range-in-month.html
3. add or change different dates to disable the dates on the calendar
4. check if month labels are labeled on the first enabled date of the month regardless of where the disabled dates in each month

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Comment by edinfor** [comment]
should I add the change log for 4.55.0 fixes?

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,7 @@
 - `[Modal]` Fixed a bug where events are not properly called when calling stacked dialogs. ([#5471](https://github.com/infor-design/enterprise/issues/5471))
 - `[Timepicker]` Fixed a bug where the chinese time format doesn't render correctly after selecting time and periods (AM/PM). ([#5420](https://github.com/infor-design/enterprise/issues/5420))
 - `[Tree]` Fixed an issue where lengthy node text doesn't wrap to lines and cuts off. ([#5499](https://github.com/infor-design/enterprise/issues/5499))
+- `[Calendar]` fixed month label not set on first enabled date of the month. ([#5581](https://github.com/infor-design/enterprise/issues/5581))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:34]:
```suggestion
- `[Calendar]`Fixed month label not set on first enabled date of the month. ([#5581](https://github.com/infor-design/enterprise/issues/5581))
```

**Review by tmcconechy — COMMENTED** [review]
@edinfor this seems good but could you update the steps on this PR on how to test? Im not certain what to look for here when trying it out

File: docs/CHANGELOG.md (hunk 1 above)
Comment (ericangeles) [reply docs/CHANGELOG.md:34]: To add to this, can you arrange the log alphabetically? @edinfor

**Comment by tmcconechy** [comment]
@edinfor can you update the PR steps and fix the log. not sure how to test this. Im also making the 4.55 release so if you want it in that we need to do this today to get this in.

**Comment by edinfor** [comment]
just did, please review

**Comment by tmcconechy** [comment]
@edinfor still not seeing anything in the `**Steps necessary to review your pull request (required)**:` section.
For example:

1. go to page
2. try to ...
3. notice ...
4. ect...

**Comment by edinfor** [comment]
updated pr to include test steps

**Review by tmcconechy — APPROVED** [review]

**Review by EdwardCoyle — APPROVED** [review]

**Pull request merged**